src/FormModel.php

```diff
@@ -262,26 +262,28 @@ private function addErrors(array $items): void
      *
      * @return array list of attribute types indexed by attribute names.
      */
-    private function attributes(): array
+    private function collectAttributes(): array
     {
         $class = new \ReflectionClass($this);
-        $this->attributes = [];
+        $attributes = [];
 
         foreach ($class->getProperties() as $property) {
-            if (!$property->isStatic()) {
-                $type = (new \ReflectionProperty($property->class, $property->name))->getType();
-
-                if ($type !== null) {
-                    $this->attributes[$property->getName()] = $type->getName();
-                } else {
-                    throw new \InvalidArgumentException(
-                        "You must specify the type hint for \"$property->class\" class."
-                    );
-                }
+            if ($property->isStatic()) {
+                continue;
```

Comment:
You are adding a mutant, just check that it is not static.

Reply:
What?

Reply:
Likely it's about mutation testing and Infection.

Reply:
https://infection.github.io/guide/mutators.html#Loop

Reply:
Foreach and for produces mutations too. We won't use it too?

Reply:
![image](https://user-images.githubusercontent.com/6815714/86167640-9405a680-bb1f-11ea-8bdc-d89ba5262cd6.png)
I've fixed it.